code/modules/admin/verbs/adminhelp.dm

```diff
@@ -256,19 +271,28 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 /datum/admin_help/proc/MessageNoRecipient(msg)
 	var/ref_src = "[REF(src)]"
 	//Message to be sent to all admins
-	var/admin_msg = "<span class='adminnotice'><span class='adminhelp'>Ticket [TicketHref("#[id]", ref_src)]</span><b>: [LinkedReplyName(ref_src)] [FullMonty(ref_src)]:</b> <span class='linkify'>[keywords_lookup(msg)]</span></span>"
-
-	AddInteraction("<font color='red'>[LinkedReplyName(ref_src)]: [msg]</font>")
+	var/admin_msg
+	if(is_mentor_ticket) //DeepSpace13 - Mentors.
+		admin_msg = "<span class='adminnotice'><span class='nicegreen'><b>Mentor Ticket [TicketHref("#[id]", ref_src)]</span><b>: [LinkedReplyName(ref_src)] [FullMonty(ref_src)]:</b> <span class='linkify'>[keywords_lookup(msg)]</span></span></b>"
+	else
+		admin_msg = "<span class='adminnotice'><span class='adminhelp'>Admin Ticket [TicketHref("#[id]", ref_src)]</span><b>: [LinkedReplyName(ref_src)] [FullMonty(ref_src)]:</b> <span class='linkify'>[keywords_lookup(msg)]</span></span>"
+	AddInteraction("<font color='#ff8c8c'>[LinkedReplyName(ref_src)]: [msg]</font>")
 
 	//send this msg to all admins
 	for(var/client/X in GLOB.admins)
 		if(X.prefs.toggles & SOUND_ADMINHELP)
-			SEND_SOUND(X, sound('sound/effects/adminhelp.ogg'))
+			if(is_mentor_ticket) //DeepSpace13 - Mentors
+				SEND_SOUND(X, sound('DS13/sound/effects/mentorhelp.ogg'))
+			else
+				SEND_SOUND(X, sound('sound/effects/adminhelp.ogg'))

```

Comment:
This changes it for the admin bwoink but not for the bwoink the player gets when he has an answer to his mentor help

Reply:
Yeah hearing data's voice (the mentorhelp sound: "How may I assist you?") would get tiresome. Perhaps another sound is in order?

Reply:
The same line but said by the Borg one time on two.